modules/web/src/app/node-data/basic/provider/kubevirt/component.ts

```diff
@@ -662,13 +662,18 @@ export class KubeVirtBasicNodeDataComponent
   private _init(): void {
     if (this._initialData) {
       this.form.get(Controls.Memory).setValue(parseInt(this._initialData.memory) || this._defaultMemory);
-      this.form.get(Controls.CPUs).setValue(parseInt(this._initialData.cpus) || this._defaultCPUs);
       this.form.get(Controls.PrimaryDiskSize).setValue(this._initialData.primaryDiskSize);
       this.form.get(Controls.PrimaryDiskOSImage).setValue(this._initialData.primaryDiskOSImage);
       this.form.get(Controls.NodeAffinityPreset).setValue(this._initialData.nodeAffinityPreset?.Type);
       this.form.get(Controls.NodeAffinityPresetKey).setValue(this._initialData.nodeAffinityPreset?.Key);
       this.nodeAffinityPresetValues = this._initialData.nodeAffinityPreset?.Values || [];
 
+      if (!_.isEmpty(this._initialData.vcpus)) {
```

Comment:
No need to use `isEmpty` here, this can simply be a nil/empty check `this._initialData.vcpus?.cores`

Reply:
@soer3n please revert this and most likely we are gonna revert any UI changes